src/circleci/rollcage/core.clj

```diff
@@ -187,3 +188,24 @@
 (def warning  (partial notify "warning"))
 (def info     (partial notify "info"))
 (def debug    (partial notify "debug"))
+
+(defn rollbar-reporter
+  "Returns a rollbar reporter function that takes an exception and
+  reports it to rollbar with the given params."
+  [rollcage-client params]
+  (infof "rollbar reporting configured for %s" params)
+  (fn [^Throwable ex]
+    (try
```

Comment:
I think the try/catch here belongs in Rollcage itself - otherwise every client if going to have to implement it. We should change it so that it doesn't throw exceptions. How about we add try/catch to `send-item`? We will need to think about the return value of `send-item`.